### DISCUSSION TITLE
Different fine tuning values for different RX

### DISCUSSION THREAD

**Discussion by ubo2021** [pascallanger/DIY-Multiprotocol-TX-Module#572] [Q&A]
In my understanding of documentation of DIY MPM TX Module, the fine tuning adjust the TX-HF-module once for each protocol. In my case I have 3 RX with frsky X2 LBT protocol which all have  different fine tuning values:
RM168 fine tuning: -6
frsky GRX8 fine tuning: 17
frsky RX6R fine tuning: 12

Maybe that I misunderstood the docu?

regards,
Ulli

**Comment by pascallanger** [comment]
Usually clones like the rm168 do an auto tune but for it to give the same value as the others you should set the same tuning value that you have gathered from the genuine RX (which are nearly the same as you've written) before binding.
Anyway it's good practice to do the RF tune for every RX.

**Comment by ubo2021** [comment]
Thanks for the explanation. So I will go ahead in fine tuning for every RX. :)
